src/mds/Server.cc

```diff
@@ -349,10 +349,17 @@ void Server::handle_client_session(MClientSession *m)
 	m->put();
 	return;
       }
+      // This looks as if we are getting a seq that is higher than
+      // expected.  I didn't notice any "wrap" handling, but the seqn is
+      // 64-bits ...
```

Comment:
I prefer to avoid uncertain "..." comments like this: this kind of creates more doubt than it removes for the developer reading this years down the line (also the "I" could be anyone when you're reading it later).  I like nice factual statements like "The client message is reporting a higher seq than the server's session state expects, can only happen on bug".